src/main/java/loanbook/model/ModelManager.java

```diff
@@ -105,6 +105,12 @@ public void setBikes(List<Bike> bikes) {
         indicateLoanBookChanged();
     }
 
+    @Override
+    public void resetBikes() {
+        setBikes(Collections.emptyList());
+        // Change has already been indicated in the above command.
```

Comment:
This comment here seems ambiguous. I am not sure what it is supposed to mean. Perhaps we can remove this?